common/util.c

```diff
@@ -138,3 +138,17 @@ bool parse_boolean(const char *boolean, bool current) {
 	// All other values are false to match i3
 	return false;
 }
+
+enum wlr_direction opposite_direction(enum wlr_direction d) {
+	switch (d) {
+		case WLR_DIRECTION_UP:
+			return WLR_DIRECTION_DOWN;
+		case WLR_DIRECTION_DOWN:
+			return WLR_DIRECTION_UP;
+		case WLR_DIRECTION_RIGHT:
+			return WLR_DIRECTION_LEFT;
+		case WLR_DIRECTION_LEFT:
+			return WLR_DIRECTION_RIGHT;
+	}
+	return 0;
```

Comment:
`assert(false)`?

Reply:
Done. Note that we can't use `sway_assert` here because this is common code and it can't link to `sway_terminate`.